composer.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "php-tuf/php-tuf",
+    "description": "PHP implementation of The Update Framework (TUF)",
+    "type": "library",
+    "require-dev": {
+        "phpunit/phpunit": "^4.8",
```

Comment:
We discussed that, although we want the client to support down to PHP 5.3 for various projects' legacy versions, we're probably better off writing the tests in a modern, secure, and forward-compatible version of PHPUnit to start. So I'd recommend PHPUnit 8 or 9 here as a dev dependency, and then backport the test suite to PHPUnit 4 as needed later. (We can still support PHP 5 for prod.)

Reply:
The issue is that we can't run CI then and certify that we support PHP 5.3+ if we don't also use older versions of PHP Unit